tests/api/utils/types.py

```diff
@@ -406,6 +419,41 @@ def __init__(self, **kwargs):
         ]
 
 
+class GenericForeignKeyList(BaseType):
```

Comment:
seems like you can use `return_value` func

Reply:
I guess it's kind of parallel thing, more of a wrapper, not a data receiver, so not sure how to use it instead of `payload` workaround